evergreen/fetch-csfle-library.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
```

Comment:
see formatting of other sh files. In 2 words, we add a header to such files that describes input and output variables

Reply:
Done